### PR TITLE
Refactor setName & setDisplayName in WizardHeaderWithDisplayNameAndName

### PR DESCRIPTION
**Pull request by ashklianko**
…content #2916

### CODE REVIEW

File: src/main/resources/assets/admin/common/js/app/wizard/WizardHeaderWithDisplayNameAndName.ts [hunk 1]
```diff
@@ -104,9 +104,13 @@ export class WizardHeaderWithDisplayNameAndName
         this.autoGenerationEnabled = value;
     }
 
-    setDisplayName(value: string, isGeneratedValue?: boolean) {
-        this.displayNameEl.setValue(value.trim());
-        this.displayNameEl.toggleClass(WizardHeaderWithDisplayNameAndName.GENERATED_CLASS, isGeneratedValue);
+    setDisplayName(value: string, silent?: boolean) {
```
Comment (alansemenov) [inline src/main/resources/assets/admin/common/js/app/wizard/WizardHeaderWithDisplayNameAndName.ts:107]: Value of `silent` parameter is never used.

File: src/main/resources/assets/admin/common/js/app/wizard/WizardHeaderWithDisplayNameAndName.ts [hunk 2]
```diff
@@ -205,8 +209,8 @@ export class WizardHeaderWithDisplayNameAndName
             !this.ignoreGenerateStatusForName && this.isNameEmptyOrEqualDisplayName(this.displayNameEl.getValue()));
     }
 
-    setName(value: string) {
-        this.nameEl.setValue(value);
+    setName(value: string, silent?: boolean) {
```
Comment (alansemenov) [inline src/main/resources/assets/admin/common/js/app/wizard/WizardHeaderWithDisplayNameAndName.ts:212]: Codacy complains about missing return type on this one.

**Review by alansemenov — CHANGES_REQUESTED** [review]

File: src/main/resources/assets/admin/common/js/app/wizard/WizardHeaderWithDisplayNameAndName.ts (hunk 1 above)
Comment (ashklianko) [reply src/main/resources/assets/admin/common/js/app/wizard/WizardHeaderWithDisplayNameAndName.ts]: My bad, fixed

File: src/main/resources/assets/admin/common/js/app/wizard/WizardHeaderWithDisplayNameAndName.ts (hunk 2 above)
Comment (ashklianko) [reply src/main/resources/assets/admin/common/js/app/wizard/WizardHeaderWithDisplayNameAndName.ts]: Fixed

**Review by alansemenov — APPROVED** [review]

**Pull request merged**